src/foam/u2/wizard/agents/AnalyticEventsAgent.js

```diff
@@ -59,6 +59,10 @@ foam.CLASS({
             // This warnings happen too frequently to be useful
             return;
           }
+
+          if ( a.length == 1 && a[0] === '' )
+            a[0] = new Error("Empty console error log").stack;
```

Comment:
We are using analytics to report errors from clients?

Reply:
We are logging console.error reported in/by the wizard as analytics events because there were no server-side log, it allows us to learn about the users experience going through the iframe flow and where the drop-off are.

Reply:
We cannot continue like this.